course/templatetags/event_grade.py

```diff
@@ -0,0 +1,20 @@
+from django import template
+from course.models.models import UserQuestionJunction
+
+register = template.Library()
+
+
+@register.filter
+def total_event_grade(event, user):
```

Comment:
This is really inefficient. It will slow than the website as the number of uqjs are a lot.
I recommend using aggregate functions you learned in 304.
also you start the filter on user.question_junctions rather than starting from the whole table.